hedera-node/hedera-mono-service/src/main/java/com/hedera/services/ServicesState.java

```diff
@@ -223,10 +207,6 @@ public void init(
                         vmFactory.apply(JasperDbBuilder::new),
                         accountMigrator);
             }
-            if (trigger == RESTART && isUpgrade) {
-                app.workingState().updateFrom(this);
-                app.treasuryCloner().ensureTreasuryClonesExist();
-            }
```

Comment:
This is needed for migration of some release to make sure treasury clones are created ?

Reply:
True, any state after release 0.28 will already have all the treasury clones.